.github/workflows/docker.build.yml

```diff
@@ -56,4 +54,43 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+      #Staging: Create pull request on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on staging
+        uses: actions/checkout@v3
+        with:
+          repository: wmde/wbaas-deploy
+          branch: deploy-to-staging-${{ steps.docker_meta.outputs.tags }}
+          committer: wmdebot <github-bots@wikimedia.de>
+          author: ${{ github.actor }} <github-bots@wikimedia.de>
+      # Make changes to pull request here
+      - run: |
+          get_tag=$(echo ${{ steps.docker_meta.outputs.tags }} | cut -d':' -f2)
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/local/ui.values.yaml.gotmpl
+          yq -i '.image.tag=$get_tag' k8s/helmfile/env/staging/ui.values.yaml.gotmpl
+      - uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: Staging+Local:Deploy new UI image ${{ steps.docker_meta.outputs.tags }}
+          title: 'Staging+Local: Deploy new UI image ${{ steps.docker_meta.outputs.tags }}'
+          labels: automated pr
+          token: ${{ secrets.WBAAS_DEPLOY_TOKEN }}
+      #Production: Create pull request on deploy repo from a merged PR
+      -
+        name: create pull request for deploy on production
+        uses: actions/checkout@v3
```

Comment:
The repository has already been checked out in one of the steps above. Is it safe/expected to run the action again in order to reset the repo state to current `main`?

Reply:
Looks like it is safe: https://github.com/actions/checkout/issues/33